### ISSUE TITLE
Priority inbox does not synchronize correctly

### ISSUE THREAD

**Issue by thstyl2000** [nextcloud/mail#4834]
### Expected behavior
Examples:

1. New mail arrives, browser sends notification and new mail appears in the frontend. (PI/All inboxes)
2. While being in the priority inbox view, marking a message from the "Other" list as favorite, makes it appear in the "Favorites" group.

### Actual behavior

1. New mail arrives, browser sends notification and new mail does not appear in PI, only in "All Inboxes".
2. While being in the priority inbox view, marking a message from the "Other" list as favorite, makes it appear in the "Favorites" group only after refresh.
3. Marking several messages from the "Others" list as important does not persist after refresh.

### Mail app

**Mail app version:** 1.9.4.

**Mailserver or service:** IMAP

### Server configuration

**Operating system**: Truenas 12.2

**Web server:** Apache

**Database:** MariaDB

**PHP version:** 7.3

**Nextcloud Version:** 19.0.8
#### Client configuration
**Browser:** Chrome 89, Firefox 86

**Operating system:** Kubuntu

**Comment by thstyl2000** [comment]
Possible duplicate of #4925?

**Comment by ChristophWurst** [comment]
yep
